iotfunctions/anomaly.py

```diff
@@ -309,9 +294,7 @@ def execute(self, df):
             dfe_orig = dfe_orig.reset_index(level=[0]).sort_index()
 
             # minimal time delta for merging
-            mindelta = dfe_orig.index.to_series().diff().min()
-            if mindelta == dt.timedelta(seconds = 0) or pd.isnull(mindelta):
-                mindelta = pd.Timedelta('5 seconds')
+            mindelta = minDelta(dfe_orig)
```

Comment:
naming convensions. even for methods. example: metod_name()

Reply:
so no camelCase ?

Reply:
renamed